### PR TITLE
Update minutes.md

### PR DESCRIPTION
**Pull request by ferrieux**
A few important details forgotten in the minutes

### CODE REVIEW

File: interim-18-09/minutes.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Demonstration of the spin bit graph
 
 Discussion of the graphs -- these just show the output of the spin bit measurement, but they don’t compare to any ground truth
 
-Orange is experimenting with three bits, but QUIC has only done privacy analysis of the single Spin Bit. That privacy analysis was done by a design team and then accepted by the working group - the other two bits must also be analyzed and accepted in order to go forward. 
+Orange is experimenting with three bits (spin bit for RTT + two bits for segmental loss), but QUIC has only done privacy analysis of the single Spin Bit. That privacy analysis was done by a design team and then accepted by the working group - the other two bits must also be analyzed and accepted in order to go forward (kinda chicken-and-egg, right ?)
```
Comment (mnot) [inline interim-18-09/minutes.md]: "kinda chicken-and-egg, right" seems like an aside to the reader, and doesn't represent the discussion. Please remove.

**Comment by ferrieux** [comment]
Yes, an aside meant to convey 1% of the frustration that the remark prompted. Guess I have no choice. Edited out.

File: interim-18-09/minutes.md (hunk 1 above)
Comment (ferrieux) [reply interim-18-09/minutes.md]: Yes, an aside meant to convey 1% of the frustration that the remark prompted. Guess I have no choice. Edited out.

**Comment by ferrieux** [comment]
Anything else preventing merge ?

**Comment by mnot** [comment]
Just flying from the US to Australia. Thanks.

**Comment by ferrieux** [comment]
Thank you Mark.

**Pull request merged**